.docs/content/1.concepts/9.mongodb.md

```diff
@@ -0,0 +1,41 @@
+# MongoDB client connection problem
+
+[MongoDB](https://www.mongodb.com/) is a document-oriented database used to build highly available and scalable applications. MongoDB solution is for high-volume data storage and uses collections and documents, unlike traditional databases. It has been noticed that when a client is connected to MongoDB, multiple connections are created. This document intends to explain why.
+
+## MongoDB in ArmoniK
+MongoDB is used in ArmoniK as database to store the metadata of the tasks. Each compute compute must connect to MongoDB database. We have in ArmoniK several connections to mongoDB per pod. The number of connections often varies between 3 and 4 or more per compute pod.  We also noticed multiple initializations of the client in ArmoniK but this will be investigated in another PR independently. 
+
+## Mongosh output interpretation
+We use mongo-compass to connect and monitor the client connection to MongoDB. The **db.serverStatus().connections** command is used in Mongosh to display the status of the client.
+
+```shell
+connections : {
+   current : <num>,
+   available : <num>,
+   totalCreated : <num>,
+   active : <num>,
+   threaded : <num>,
+   exhaustIsMaster : <num>,
+   exhaustHello : <num>,
+   awaitingTopologyChanges : <num>,
+   loadBalanced : <num>
+},
+```
+
+We will not discuss all the output values only relevant ones, the full description of this command can be found in [mongosh documentation](https://www.mongodb.com/docs/manual/reference/command/serverStatus/#connections).
+- connections.current
+The number of incoming connections from clients to the database server. This number includes the current shell session. Consider the value of connections.available to add more context to this datum.
+
+- connections.active
+The number of active client connections to the server. Active client connection refers to client connections that currently have operations in progress.
+
+These two prameters give the number of current connections to mongoDB and active connections. The current connections are to high compared to the number of clients we are running. For a deployment with a default (3) number of compute pods, the curent connections are about 42 while the active connections is around 10, which is still high since we have only one client running. Althought, our client have multiple service that may open connections, but still below the active connection number shown in mongo shell. The reason of the extra connections are discussed in the next section.
+
+## MongoDB creates multiple connections: Why
+The active client connections seen from the value *connections.active* is not the real connection number to mongoDB. This number is greater than the actual connection number because MongoDB creates additional connections for future clients that may potentially connect to the database.  
```

Comment:
```suggestion
The active client connections seen from the value `connections.active` is not the real connection number to MongoDB. This number is greater than the actual connection number because MongoDB is creating additional connections for future clients that may potentially connect to the database.  
```

Reply:
Done !